charts/mlflow-quickstart/Chart.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v2
+name: mlflow-quickstart
+description: A Helm chart for Kubernetes
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+appVersion: "2.2.1"
+
+dependencies:
+  - name: mlflow
+    repository: "file://../mlflow"
+    version: 0.1.0
+  - name: minio
+    repository: https://charts.min.io/
+    version: 5.0.7
+    condition: minio.enabled
+  - name: postgresql
+    repository: https://charts.bitnami.com/bitnami
+    version: 12.2.1
+    condition: posegresql.enabled
```

Comment:
Happen to notice a typo here. Should be: 
`    condition: postgresql.enabled`

Reply:
```suggestion
    condition: postgresql.enabled
```